### DISCUSSION TITLE
Documentation on print summary output

### DISCUSSION THREAD

**Discussion by AruunnAnanda** [google/lightweight_mmm#123] [Q&A] [answered]
Hi there,

Can I find somewhere what each of those names mean? Of course a few are self-explanatory, but not all. 

<img width="264" alt="image" src="https://user-images.githubusercontent.com/107944288/208060427-3f7bd859-197e-4dd4-b3e3-2b5febe3c0a0.png">

I tried to compare with equation parameters on this [page](https://lightweight-mmm.readthedocs.io/en/latest/models.html), but couldn't find a one to one comparison. 

It would be great and convenient if we had a documentation explaining what each of those names mean if we don't have one. 

Thank you!

**Comment by greenfrog555** [comment, marked as answer]
Hi AruunnAnanda, 

This is the [page](https://lightweight-mmm.readthedocs.io/en/latest/custom_priors.html) that you can reference about detailed definitions of those parameters and their prior distributions. 

Let me know whether you have any other questions.
